src/Microsoft.Health.Dicom.Core/Features/Telemetry/UpdateMeter.cs

```diff
@@ -0,0 +1,23 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+public sealed class UpdateMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    public UpdateMeter()
+    {
+        _meter = new Meter($"{OpenTelemetryLabels.PlatformBaseMeterName}.Update", "1.0");
+        UpdatedInstances = _meter.CreateCounter<int>(nameof(UpdatedInstances), description: "Count of instances updated successfully");
```

Comment:
Should the metric just be called `Count` given the [OTel naming conventions](https://opentelemetry.io/docs/reference/specification/metrics/semantic_conventions/#use-count-instead-of-pluralization)

Reply:
Or if we anticipate we want to count other things like studies and series, then perhaps just `"instances"`?

Reply:
Discussed offline.